<launch>	

    <arg name="robot_name" default="robot1" />
    <arg name="robot_type" default="freight"/>
 
    <group ns="robot1">
        <node pkg="move_base" type="move_base" name="move_base" respawn="true" output="screen">
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/common/costmap_common.yaml" command="load" ns="global_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/$(arg robot_type)/costmap_common.yaml" command="load" ns="global_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/common/costmap_common.yaml" command="load" ns="local_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/$(arg robot_type)/costmap_common.yaml" command="load" ns="local_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/common/costmap_local.yaml" command="load" ns="local_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/$(arg robot_type)/costmap_local.yaml" command="load" ns="local_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/common/costmap_global.yaml" command="load" ns="global_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/$(arg robot_type)/costmap_global.yaml" command="load" ns="global_costmap" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/common/move_base.yaml" command="load" />
    	   <rosparam file="$(find warehousetest)/config/navigation/fetch_freight/$(arg robot_type)/move_base.yaml" command="load" />
    	   <param name="global_costmap/robot_base_frame" value="$(arg robot_name)_tf/base_link"/>
    	   <param name="local_costmap/robot_base_frame" value="$(arg robot_name)_tf/base_link"/>
    	   <param name="local_costmap/global_frame" value="$(arg robot_name)_tf/odom" />
    	   <remap from="map" to="/map" />
        </node>
    </group>
</launch>